frontend/src/layout/navigation-3000/navigationLogic.tsx

```diff
@@ -329,6 +335,30 @@ export const navigation3000Logic = kea<navigation3000LogicType>([
                             icon: <IconDashboard />,
                             logic: isUsingSidebar ? dashboardsSidebarLogic : undefined,
                             to: isUsingSidebar ? undefined : urls.dashboards(),
+                            sideAction: dashboardsLoading
```

Comment:
Personally I find this somewhat annoying. Having a spinner there draws unnecessary attention to something that is rarely clicked.

Could we not put the loading state in the overlay instead?

Reply:
Makes sense. I moved the loading state into the overlay instead.

Though I saw @corywatilo reacted with a 👎 -emoji. Does this mean this should not go in at all? Or just in a different version?

Reply:
Also also one thing I just thought of - it isn't clear at first glance that they are pinned dashboards... Does that menu dropdown need a little title like it used to have?

Reply:
Oh, definitely needs a title. Added.
<img width="264" alt="Screenshot 2024-01-15 at 11 01 17" src="https://github.com/PostHog/posthog/assets/1851359/351cbb2f-9bd3-4f88-8398-8f08aa4ff9e6">
<img width="260" alt="Screenshot 2024-01-15 at 11 01 36" src="https://github.com/PostHog/posthog/assets/1851359/5d75e25e-e488-41f5-8ee1-798a23de2f75">